docs/index.html

```diff
@@ -0,0 +1,217 @@
+
+  <header id="head">
+      <img src='./assets/images/245892900001212.png' height=100 style="float:left; padding-right: 15px; position: relative; padding-left: 75px; " />
+      <h2 style='color: #16b9d4'>Hedwig</h2>
+  </header>
+
+  <div id="app">
+    <div id="stage">
+      <nav id="nav" class="hxNav">
+        <!-- Product Navigation -->
+        <a href="#getting-started">Getting Started</a>
+        <a href="#install">Install</a>
+        <a href="#how-to">How To Use</a>
+
+        <hx-disclosure aria-controls="section-1">
+          Components
+          <hx-icon class="toggle-icon" type="angle-down"></hx-icon>
+        </hx-disclosure>
+        <hx-reveal id="section-1" open>
+          <a href="#cpu-system-usage">cpu system usage</a>
+        </hx-reveal>
+        <!-- end:Product Navigation -->
+      </nav>
+
+      <!-- Accessibility: [role="main"] must be present on main#content -->
+      <main role="main" id="content">
+        <hx-panel class="hxSpan-12-xs">
+          <hx-panelbody class="hxBox hxMd">
+            <div>
+              <p>
+                Hedwig is a component library for graphing monitoring data for Rackspace.
+              </p>
+              <p>
+                It is based on web components, so it can fit inside of any web UI application.
+              </p>
+            </div>
+            <br><br>
+            <h3 id='getting-started'>How to get started</h3>
+            <div>
+              To get started simply install Hedwig into your application
+            </div>
+            <br><br>
+            <h3 id='install'>Installing Hedwig</h3>
+            <div id='install'>
+              <p>
+                Install using npm
+              </p>
+              <br>
+              <pre><code class="bash">npm install rackspace/hedwig</code></pre>
+            </div>
+            <br><br>
+            <div id='how-to'>
+              <h3>How To Use</h3>
+              <p>
+                Hedwig uses <a href='https://www.webcomponents.org'>web components</a> so it is quite easy to use.
+              </p>
+              <p>
+                Just add a component tag to your page and add any data-* attributes for configuration
+              </p>
+              <br>
+              <pre><code class='html'>&lt;system-usage data-height="200" data-width="400" data={{ someBoundData }} &gt;&lt;/system-usage&gt;</code></pre>
+              <br>
+              <p>This will render the graph, taking in the data and mapping according to the components perscrabed data format.</p>
+            </div>
+            <br><br>
+            <hr>
+            <br><br>
+            <div id='components'>
+              <h2>Components</h2>
+            </div>
+            <br><br>
+            <div id='cpu-system-usage'>
+              <h3>CPU System Usage</h3>
+              <div>
+                <p>
+                  Description:<br><br>
+                  Dispalys the percentage of the CPU used by the system over time
```

Comment:
I finally found a use for the github suggestion feature :)

```suggestion
                  Displays the percentage of the CPU used by the system over time
```

Reply:
hehe I will fix that.

Thank you, we will have to make changes as we go.